component-overview/webapp/styles/sb1ex-page.css

```diff
@@ -1,6 +1,5 @@
 /* stylelint-disable selector-class-pattern */
 .sb1ex-page {
-    height: 100vh;
```

Comment:
Det her var banditen som gjorde att det ikek funket

Reply:
Kanskje den kan erstattes med `min-height: 100%;` eller noe sånt i stedet for å fjernes helt? Når `.sb1ex-page` ikke er 100% høy funker ikke toggling av bakgrunnsfarge helt som det skal.

<kbd><img width="300" alt="image" src="https://github.com/SpareBank1/designsystem/assets/463847/d63e836f-1710-42a7-8490-d61fa44ef4be"></kbd>

Reply:
min-height: 100% ser ut som det virker som forventet, enig!

Reply:
`min-height: 100% ` kjørde jag på. Funker bra tror jag